Generator/Commands/ContainerGenerator.php

```diff
@@ -192,6 +192,82 @@ public function getUserInputs()
                 'task'      => 'Delete' . $model . 'Task',
             ],
         ];
+
+        if ($ui == 'web') {
+            $routes = [
+                [
+                    'stub'      => 'GetAll',
+                    'name'      => 'GetAll' . $models,
+                    'operation' => 'index',
+                    'verb'      => 'GET',
+                    'url'       => $url,
+                    'action'    => 'GetAll' . $models . 'Action',
+                    'request'   => 'GetAll' . $models . 'Request',
+                    'task'      => 'GetAll' . $models . 'Task',
+                ],
+                [
+                    'stub'      => 'GetOne',
+                    'name'      => 'Get' . $model . 'ById',
+                    'operation' => 'show',
+                    'verb'      => 'GET',
+                    'url'       => $url . '/{id}',
+                    'action'    => 'Get' . $model . 'ById' . 'Action',
+                    'request'   => 'Get' . $model . 'ById' . 'Request',
+                    'task'      => 'Get' . $model . 'ById' . 'Task',
+                ],
+                [
+                    'stub'      => null,
```

Comment:
I don't think it's good to pass `null` values on specific fields.. because there is actually no `null` stub defined.. This would confuse others.. So rather make a new field in the array and check on this field.. **or**, just use the `$ui`, because this field already indicates what to do, right?